### PR TITLE
feat: simplifies background and make login responsive

### PR DESCRIPTION
**Pull request by sdilauro**
- Change Decentraland Auth for DCLExplorer Auth in readme
- Delete link to desktop client download
- Simplified the design to be able to iterate more quickly. It can be embellished when we have it fully responsive and functional.

Example mobile view:
Image

Example desktop view:
Image

Example screen with width < 240px:
Image

### CODE REVIEW

File: src/assets/images/facebook.svg
Comment (kuruk-mm) [inline src/assets/images/facebook.svg:1]: can you use a more rounded facebook logo? Like what they are using right now
image

File: src/components/Connection/Connection.module.css [hunk 1]
```diff
@@ -4,6 +4,8 @@
   color: #ffffff;
   display: flex;
   flex-direction: column;
+  padding-top:40px;
+  padding-bottom:40px;
```
Comment (kuruk-mm) [inline src/components/Connection/Connection.module.css]: add task to the backlog: add linter to decentraland-auth

```suggestion
  padding-top: 40px;
  padding-bottom: 40px;
```

File: src/components/Connection/Connection.spec.tsx [hunk 2]
```diff
@@ -15,8 +16,8 @@ function renderConnection(props: Partial<ConnectionProps>) {
       i18n={{
         title: 'Unlock Your Virtual World.',
         subtitle: 'Access and start exploring.',
-        accessWith: option => `Access with ${option}`,
-        connectWith: option => `Connect with ${option}`,
+        accessWith: option => `Access with ${option.toString().replace('-', '')}`,
+        connectWith: option => `Connect with ${option.toString().replace('-', '')}`,
```
Comment (kuruk-mm) [inline src/components/Connection/Connection.spec.tsx]: I didn't like this solution. You can move it to a function and give a logic like:

```
function getWalletNameFromOption(option: String) {
  if (option === "wallet-connect") {
    return "Wallet Connect"
  }
  return option
}
```

and then apply it to the option like `getWalletNameFromOption(option)`

**Review by kuruk-mm — CHANGES_REQUESTED** [review]

File: src/assets/images/facebook.svg
Comment (sdilauro) [reply src/assets/images/facebook.svg:1]: done

File: src/components/Connection/Connection.spec.tsx (hunk 2 above)
Comment (sdilauro) [reply src/components/Connection/Connection.spec.tsx]: done

File: src/components/Connection/Connection.types.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 export enum ConnectionOptionType {
   METAMASK = 'metamask',
   DAPPER = 'dapper',
+  FACEBOOK = 'facebook', // Add facebook
```
Comment (kuruk-mm) [inline src/components/Connection/Connection.types.ts]:
```suggestion
  FACEBOOK = 'facebook',
```

File: src/components/Connection/Connection.tsx [hunk 4]
```diff
@@ -54,7 +56,10 @@ const Secondary = ({
   onConnect: (wallet: ConnectionOptionType) => unknown
 }) => (
   <div className={styles.showMoreSecondaryOptions} data-testid={testId}>
-    {options.map(option => (
+    {
+    options
+    .filter(option => option !== 'wallet-connect') // This is to add wallet-connect like main button and don't repeat it at show more section
+    .map(option => (
```
Comment (kuruk-mm) [inline src/components/Connection/Connection.tsx]: this is wrong, the wallet-connect should be removed here:
https://github.com/leanmendoza/decentraland-auth/blob/b232e9060b508714a66cb609a04ac57dace4cadd/src/components/Pages/LoginPage/LoginPage.tsx#L68-L76

**Review by kuruk-mm — CHANGES_REQUESTED** [review]

File: src/components/Connection/Connection.spec.tsx (hunk 2 above)
Comment (sdilauro) [reply src/components/Connection/Connection.spec.tsx]: added const ConnectionOptionTitle

File: src/components/Connection/Connection.tsx (hunk 4 above)
Comment (sdilauro) [reply src/components/Connection/Connection.tsx]: Yes. I will fix it

File: src/components/Connection/Connection.types.ts (hunk 3 above)
Comment (sdilauro) [reply src/components/Connection/Connection.types.ts]: Yes. The comment was deleted

File: src/components/Connection/Connection.module.css (hunk 1 above)
Comment (sdilauro) [reply src/components/Connection/Connection.module.css]: Fixed manually.

**Review by kuruk-mm — APPROVED** [review]

**Review by leanmendoza — APPROVED** [review]
LGTM!

**Pull request merged**